store/db_test.go

```diff
@@ -78,6 +78,9 @@ func createTable(db *DB, t *testing.T) {
 }
 
 func TestDBRace(t *testing.T) {
+	// TODO(sgotti) this will not find concurrenct accesses to ql db from
+	// multiple processes using multiple goroutines. A test that spawns at
```

Comment:
@sgotti I don't really understand your use of the word "process" in this context (and in the original [comment](https://github.com/coreos/rkt/pull/2391#issue-146668020)). Isn't this just a problem within a single process with multiple goroutines?